README.rst

```diff
@@ -190,9 +190,14 @@ Here is an end to end example of how to use a SageMaker Estimator:
     response = mxnet_predictor.predict(data)
 
     # Tears down the SageMaker endpoint
```

Comment:
Let's update this comment to reflect that both the endpoint and endpoint configuration are removed.